.github/workflows/test.yml

```diff
@@ -150,12 +150,10 @@ jobs:
       - name: filter out DONTCOVER
         # yamllint disable
         run: |
-          excludelist="$(find ./ -type f -name '*.go' | xargs grep -l 'DONTCOVER')"
-          excludelist+=" $(find ./ -type f -name '*.pb.go')"
-          excludelist+=" $(find ./ -type f -name '*.pb.gw.go')"
-          excludelist+=" $(find ./ -type f -path './tests/mocks/*.go')"
+          excludelist="$(find . -type f -name '*.go' | xargs grep -l 'DONTCOVER')\n"
+          excludelist+="$(find . -type f -name '*.pb.*')\n"
+          excludelist+="$(find . -type f -path './tests/mocks/*.go')"
           for filename in ${excludelist}; do
-            filename=$(echo $filename | sed 's/^./github.com\/cosmos\/cosmos-sdk/g')
```

Comment:
it's not clear to me what the fix is. Can you please elaborate in the PR description or a code comment?